playground/mocks/data/idp/idx/okta-verify-version-upgrade.json

```diff
@@ -137,7 +137,7 @@
       {
         "message": "Your response was received, but your Okta Verify version is no longer supported by your organization. To verify your identity with push notifcations, update Okta Verify to the latest version, then try again.",
         "i18n":{
-            "key": "idx.authenticator.app.method.push.force.upgrade.number_challenge",
+            "key": "oie.authenticator.app.method.push.force.upgrade.number_challenge",
```

Comment:
Updated key to have `oie` prefix to be consistent with other keys  https://oktainc.atlassian.net/browse/OKTA-381041